apps/tmux/tmux.py

```diff
@@ -0,0 +1,120 @@
+from talon import Context, Module, actions
+
+ctx = Context()
+mod = Module()
+
+ctx.matches = r"""
+tag: user.tmux
+"""
+
+setting_tmux_prefix_key = mod.setting(
+    "tmux_prefix_key",
+    type=str,
+    default="b",
```

Comment:
It would be neat if we could parse the user's default prefix key from their config automatically (but doesn't need to be done in this PR).

Reply:
I guess I can work on that if anyone is willing to approve this pr 😅